test/jit/test_slice.py

```diff
@@ -0,0 +1,101 @@
+import os
+import sys
+
+import torch
+from typing import List
+
+# Make the helper files in test/ importable
+pytorch_test_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+sys.path.append(pytorch_test_dir)
+from torch.testing._internal.jit_utils import JitTestCase
+
+if __name__ == '__main__':
+    raise RuntimeError("This test file is not meant to be run directly, use:\n\n"
+                       "\tpython test/test_jit.py TESTNAME\n\n"
+                       "instead.")
+
+# Tests that Python slice class is supported in TorchScript
+class TestSlice(JitTestCase):
```

Comment:
Consider adding a test in which `stop` is greater than the length of the iterable to check that out of bounds indices are clipped correctly.

Reply:
Done.